sdk/tests/deqp/framework/common/tcuFloatFormat.js

```diff
@@ -125,6 +125,13 @@ goog.require('framework.delibs.debase.deMath');
      };
 
      /**
+      * @return {tcuFloatFormat.YesNoMaybe}
```

Comment:
the native code: https://android.googlesource.com/platform/external/deqp/+/deqp-dev/framework/common/tcuFloatFormat.hpp#58